model/textparse/protobufparse.go

```diff
@@ -629,7 +634,7 @@ func formatOpenMetricsFloat(f float64) string {
 	case math.IsInf(f, -1):
 		return "-Inf"
 	}
-	s := fmt.Sprint(f)
+	s := string(strconv.AppendFloat(b[:0], f, 'g', -1, 64))
```

Comment:
Interesting that this is really measurably better in performance.

Now that we are acting on a bytes buffer anyway, how about folding the operation down in line 641 into this? I.e. appending the `".0"` string is much more efficient if using a mutable bytes buffer rather than string concatenation.

Reply:
Good idea, yes it should be beneficial (even though it doesn't change anything in `BenchmarkParse`  case).